recipe/doc/end_user_run_guide.md

```diff
@@ -0,0 +1,121 @@
+- Driver libraries, such as (but not limited to)
+    - `libcuda`
+    - `libnvidia-ml` library
+- Older versions of CTK contained these
+    - Documentation
+    - Samples (use to be)
+    - Demo suite (limited subset of samples)
+- GDS
+    - Missing file system components
+- fabricmanager
+- `libnvidia_nscq`
+- Imex
+- Nsight-systems
+
+### Installing Subsets of the CTK
+
+Rather than installing all of CUDA at once, users may instead install just the packages that they need. 
+For example, to install just libcublas or libcusparse one may run:
+```
+conda install -c conda-forge libcublas cuda-version=<CUDA version>
+```
+The best way to get a current listing is to run
+```
+conda install --dry-run -c conda-forge cuda cuda-version=<CUDA version>
+```
+Original build order: https://github.com/conda-forge/staged-recipes/issues/21382.
+
+### Metapackages
+
+Existing conda documentation: https://docs.nvidia.com/cuda/cuda-installation-guide-linux/#conda-installation
+
+For convenience, a number of additional metapackages are available:
+- `cuda-runtime`: All CUDA runtime libraries needed to run a CUDA application
+- `cuda-libraries`: All libraries required to run a CUDA application requiring libraries beyond the CUDA runtime (such as the CUDA math libraries) as well as packages needed to perform JIT compilation.
+- `cuda-visual-tools`: GUIs for visualizing and profiling such as Nsight Systems and Nsight Compute
+- `cuda-command-line-tools`: Command line tools for analyzing and profiling such as cupti, cuda-gdb, and Compute Sanitizer.
+- `cuda-tools`: All tools for analyzing and profiling, both GUI (includes cuda-visual-tools) and CLI (includes cuda-command-line-tools)
+
+### CUDA C++ Core Libraries (CCCL)
+
+CCCL is a special case among CUDA packages.
+Due to 1) being header-only, 2) fast-moving, and 3) independently-evolving, consumers may want a different (newer) version of CCCL than the one corresponding to their CTK version.
+Instructions on how to install a suitable CCCL package from conda can be found [in the CCCL README](https://github.com/NVIDIA/cccl/?tab=readme-ov-file#conda). (See [this issue](https://github.com/conda-forge/cuda-cccl-impl-feedstock/issues/2) for more information on the history of these packages).
+
+## conda-forge vs nvidia channel
+
+Understanding the difference between the CUDA packages on the conda-forge and nvidia channels requires a bit of history because of how the relationship has evolved over time.
+In particular, how these channels may or may not coexist will depend on the versions of CUDA that you need support for.
+
+### Pre-CUDA 12:
+
+Prior to CUDA 12, the only package available on conda-forge was the `cudatoolkit` package, a community-maintained, monolithic package containing the entire repackaged CTK.
+During the CUDA 11 release cycle, NVIDIA began maintaining a set of CUDA Toolkit packages in the nvidia channel.
```

Comment:
Use code font for the channel names so it's not confused with NVIDIA the company name. I'd match this for `conda-forge` as well, when referring to the channel.
```suggestion
During the CUDA 11 release cycle, NVIDIA began maintaining a set of CUDA Toolkit packages in the `nvidia` channel.
```

Reply:
Aligned across all files.